### DISCUSSION TITLE
Is contribution open?

### DISCUSSION THREAD

**Discussion by logeekal** [tailwindlabs/headlessui#329] [General]
hello tailwind team,

You are doing an awesome work and a headless UI for react seems like a fabulous approach.

Is the project currently open for outside contribution as well as I would like learn and contribute as this project builds up.

Thanks,

Regards,
Jatin Kathuria

**Comment by ebisbe** [comment]
I was also thinking the same.

**Comment by andreasphil** [comment]
I was wondering the same. I love the idea and would definitely be interested in contributing!

**Comment by zzhenryquezz** [comment]
Same here too
